cmd/cluster/create/runner.go

```diff
@@ -0,0 +1,57 @@
+package create
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cluster/config"
+	"sigs.k8s.io/kind/pkg/cluster/create"
+)
+
+const (
+	waitForReady = 2 * time.Minute
```

Comment:
Default is 0 so no timeout which I don't like. Locally takes under a minute for me but I'd like a bit of slack.